<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signin</class>
 <widget class="QMainWindow" name="signin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 248, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>251</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>419</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 230, 255);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 12pt &quot;宋体&quot;;
color: rgb(0,0,0);</string>
    </property>
    <property name="placeholderText">
     <string>Places input your name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>251</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>419</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 230, 255);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 12pt &quot;宋体&quot;;
color: rgb(0,0,0);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Please input your password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="login_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>324</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 20pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>欢迎登录高考成绩管理系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_signin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QMainWindow#signin QPushButton#btn_signin
{
	
	background-color: rgb(174, 255, 186);
  color: rgb(0,0,0);

border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 18pt &quot;Candara&quot;;

}

QMainWindow#signin QPushButton#btn_signin:hover
{
	background-color: rgb(120,200,150);
}</string>
    </property>
    <property name="text">
     <string>确认登录</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
